packages/sil-governance/src/routes/dashboard/components/ProposalsList.tsx

```diff
@@ -35,11 +35,19 @@ const ProposalsList = ({ filterType }: Props): JSX.Element => {
   const filter = getFilter(filterType);
   const proposals = ReactRedux.useSelector((state: RootState) => state.proposals).filter(filter);
 
+  const hasProposals = proposals.length > 0;
+
   return (
     <Block id={PROPOSALS_HTML_ID} flexGrow={1}>
-      {proposals.map((proposal, index) => (
-        <RenderedProposal key={proposal.id} proposal={proposal} index={index} />
-      ))}
+      {!hasProposals && (
+        <Block margin={2}>
+          <Typography>No proposals available</Typography>
+        </Block>
+      )}
+      {hasProposals &&
```

Comment:
This list can remain unchanged, right? For 0 elements there are 0 `<RenderedProposal` elements anyways.